### ISSUE TITLE
Why is there a slight gap in the results of my training multiple times, I fixed all possible random seeds

### ISSUE THREAD

**Issue by JiuyangDong** [isyangshu/MambaMIL#17]
![7cb5f2b9bcf9fa63d31497810846715](https://github.com/isyangshu/MambaMIL/assets/110771682/f06e90ea-96f4-48f2-8635-f9c5baccd925)
![148d8fc183340202207545ed8256d8f](https://github.com/isyangshu/MambaMIL/assets/110771682/cef8b1dc-b598-49c7-bbda-7b4322d5fb94)

**Comment by JiuyangDong** [comment]
Is this due to the randomness that Mamba brings to the table itself?

**Comment by isyangshu** [comment]
Sry for late reply.

For the instability of Mamba, you can refer to https://github.com/state-spaces/mamba/issues/137.
You can use a smaller learning rate, which may allow you to reach the same local optimum before the backpropagation divergence occurs when you run the code multiple times.
